package.json

```diff
@@ -0,0 +1,131 @@
+{
+	"name": "@natlibfi/melinda-rest-api-commons",
+	"description": "Shared modules for microservices of Melinda rest api import system",
+	"author": {
+		"name": "The National Library of Finland"
+	},
+	"keywords": [],
+	"homepage": "https://github.com/natlibfi/melinda-rest-api-commons",
+	"bugs": {
+		"url": "https://github.com/natlibfi/melinda-rest-api-commons/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git@github.com:natlibfi/melinda-rest-api-commons.git"
+	},
+	"license": "AGPL-3.0+",
+	"version": "1.0.0",
+	"main": "./dist/index.js",
+	"engines": {
+		"node": ">=12"
+	},
+	"private": true,
+	"scripts": {
+		"prepare": "npm run build",
+		"lint": "eslint ./src",
+		"lint:dev": "eslint ./src --fix",
+		"check-deps": "npm-check;exit 0",
+		"test": "npm run lint:dev && npm run test:base -- src/**/*.spec.js",
```

Comment:
You need to have `test` and `test:dev`. `test` must not modify files since it's used in building.